client/dashboard/sites/settings-repositories/connect-repository.tsx

```diff
@@ -49,17 +49,17 @@ export default function ConnectRepository() {
 				<PageHeader
 					prefix={ <Breadcrumbs length={ 3 } /> }
 					title={ __( 'Connect Repository' ) }
-					description={ __( 'Connect a GitHub repository to deploy code to your WordPress site.' ) }
+					description={ __( 'Deploy code from GitHub to your WordPress site.' ) }
 				/>
 			}
 		>
 			<Card>
 				<CardBody>
 					<ConnectRepositoryForm
-						formTitle={ __( 'Configure repository connection' ) }
+						formTitle={ __( 'Set Up Connection' ) }
```

Comment:
I think we now use sentence case:

```suggestion
						formTitle={ __( 'Set up connection' ) }
```